msteams-platform/toolkit/provision.md

```diff
@@ -1,621 +1,435 @@
 
-> [!NOTE]
-> When you add resources or capabilities to your project, `teamsfx\xxx.bicep` is regenerated, you can't customize the same. To modify the bicep files, you can use Git to track your changes to `teamsfx\xxx.bicep` files. This doesn't make you lose any changes while adding resources or capabilities to your project.
+1. Open `teamsapp.yml` and find the `aadApp/create` action.
 
-The ARM template files use placeholders for parameters. The purpose of the placeholders is to ensure that new resources can be created in a new environment. The actual values are resolved from `.fx\states\state.{env}.json` file.
+1. Find the environment variable names that store information for Azure AD app in the `writeToEnvironmentFile` property. Below is the default `writeToenvironmentFile` definition if you create projects using Teams Toolkit:
 
-### Azure AD app related parameters
+ ```yml
+  writeToEnvironmentFile:
+   clientId: AAD_APP_CLIENT_ID
+   clientSecret: SECRET_AAD_APP_CLIENT_SECRET
+   objectId: AAD_APP_OBJECT_ID
+   tenantId: AAD_APP_TENANT_ID
+   authority: AAD_APP_OAUTH_AUTHORITY
+   authorityHost: AAD_APP_OAUTH_AUTHORITY_HOST
+ ```
 
-| Parameter name | Default value placeholder | Meaning of the placeholder | How to customize |
-| --- | --- | --- | --- |
-| Microsoft 365 ClientId | {{state.fx-resource-aad-app-for-teams.clientId}} | Your app's Azure AD app client Id created during provision. | [Use an existing Azure AD app for your Teams app](#use-an-existing-azure-ad-app-for-your-teams-app). |
-| Microsoft 365 ClientSecret | {{state.fx-resource-aad-app-for-teams.clientSecret}} | Your app's Azure AD app client secret is created during provision. | [Use an existing Azure AD app for your Teams app](#use-an-existing-azure-ad-app-for-your-teams-app).  |
-| Microsoft 365 TenantId | {{state.fx-resource-aad-app-for-teams.tenantId}} | Tenant Id of your app's Azure AD app. | [Use an existing Azure AD app for your Teams app](#use-an-existing-azure-ad-app-for-your-teams-app).  |
-| Microsoft 365 OAuthAuthorityHost | {{state.fx-resource-aad-app-for-teams.oauthHost}} | OAuth authority host of your app's Azure AD app. | [Use an existing Azure AD app for your Teams app](#use-an-existing-azure-ad-app-for-your-teams-app). |
-| botAadAppClientId | {{state.fx-resource-bot.botId}} | Bot's Azure AD app client Id is created during provision. | [Use an existing Azure AD app for your bot](#use-an-existing-azure-ad-app-for-your-bot). |
-| botAadAppClientSecret | {{state.fx-resource-bot.botPassword}} | Bot's Azure AD app client secret is created during provision. | [Use an existing Azure AD app for your bot](#use-an-existing-azure-ad-app-for-your-bot). |
+1. Add values for each environment variable from step 2.
 
-### Reference environment variables in parameter files
+    1. Add below environment variables and their values to `env\.env.{env}` file.
```

Comment:
The numbering is wrong in the generated content

Reply:
![image](https://github.com/MicrosoftDocs/msteams-docs/assets/16605901/573be319-aed2-40be-874b-a7b0a5f94f13)

Reply:
Updated.